docs/_frontend/webpack.config.js

```diff
@@ -111,6 +111,7 @@ module.exports = (env, argv) => {
         SEARCH_ORIGIN:
           process.env.SEARCH_ORIGIN ||
           'https://search.modelshub.johnsnowlabs.com',
+        ORIGIN: process.env.ORIGIN || 'https://sparknlp.org',
```

Comment:
I think we could remove a default value from here.

Reply:
I would prefer to keep the frontend code the same as we have in the `johnsnowlabs` repo.

Reply:
Remove default value